### PR TITLE
Adding ability to send simple unformatted messages

### PR DESCRIPTION
**Pull request by antonsamper**

### CODE REVIEW

**Comment by antonsamper** [comment]
This fixes #18 

Let me known if you have any comments regarding the implementation

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Creates a new GoodSlack object with the following arguments:
   - `[slack]` - an object of slack overridable parameters (See [Incoming Webhooks](https://api.slack.com/incoming-webhooks))
   - `[format]` - [MomentJS](http://momentjs.com/docs/#/displaying/format/) format string. Defaults to 'YYMMDD/HHmmss.SSS'.
   - `[host]` - a string with the server hostname. - Defaults to actual hostname.
+  - `[basicLogEvent]` - a boolean to set the style of `log` events. When set to true, `log` events will be sent as text instead of attachments.
```
Comment (nakardo) [inline README.md]: Describe the default value here as in `format` and `host`.

**Comment by antonsamper** [comment]
@nakardo done 👍

**Comment by antonsamper** [comment]
hi @nakardo, are you able to merge this PR?

**Comment by nakardo** [comment]
I'm traveling and won't be able to do so unfortunately. Will do as soon as
I get back. Thanks.

Il 04 apr 2017 6:56 AM, "Anton Samper Rivaya" <notifications@github.com> ha
scritto:

hi @nakardo <https://github.com/nakardo>, are you able to merge this PR?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/nakardo/good-slack/pull/19#issuecomment-291452274>, or mute
the thread

**Comment by nakardo** [comment]
@antonsamper apologies for the delay, I've just published the new version.

**Comment by antonsamper** [comment]
@nakardo no problem, thanks for merging 👍

**Pull request merged**